files/zh-cn/games/techniques/control_mechanisms/mobile_touch/index.md

```diff
@@ -25,91 +27,91 @@ el.addEventListener("touchcancel", handleCancel);
 
 这样，在移动设备上屏幕上触摸游戏的 {{htmlelement("canvas")}} 将触发这些事件，因为我们就可以随意操控游戏（如：移动太空船）。事件如下所示：
 
-- [touchstart](/zh-CN/docs/Web/API/GlobalEventHandlers/ontouchstart) 当用户手指放在屏幕上触发。
-- [touchmove](/zh-CN/docs/Web/API/GlobalEventHandlers/ontouchmove) 当他们在屏幕上移动手指时触发。
-- [touchend](/zh-CN/docs/Web/API/GlobalEventHandlers/ontouchend) 当用户在屏幕上停止移动时触发。
-- [touchcancel](/zh-CN/docs/Web/API/GlobalEventHandlers/ontouchcancel) 触摸被取消是触发，例如当用户将他们的手指移动到屏幕之外时。
+- 当用户手指放在屏幕上触发 [touchstart](/zh-CN/docs/Web/API/Element/touchstart_event) 事件。
+- 当用户在屏幕上移动手指时触发 [touchmove](/zh-CN/docs/Web/API/Element/touchmove_event) 事件。
+- 当用户停止触摸屏幕时触发 [touchend](/zh-CN/docs/Web/API/Element/touchend_event) 事件。
+- 当触摸被取消，例如当用户将他们的手指移动到屏幕之外时触发 [touchcancel](/zh-CN/docs/Web/API/Element/touchcancel_event) 事件。
 
-> **备注：** 这篇 [touch events](/zh-CN/docs/Web/API/Touch_events) 参考文章提供了更多的实例和介绍。
+> **备注：** [触摸事件](/zh-CN/docs/Web/API/Touch_events)参考文章提供了更多的实例和介绍。
 
 ### 纯 JavaScript 示例
 
-这个实现了移动端触摸的[little demo](https://github.com/end3r/JavaScript-Game-Controls/)代码已经放到了 GibHub 上，我们下载这个示例就可以实现在移动端屏幕上移动飞船。
+这个实现了移动端触摸的[小型示例](https://github.com/end3r/JavaScript-Game-Controls/)代码已经放到了 GitHub 上，我们下载这个示例就可以实现在移动端屏幕上移动飞船。
 
-我们将两种事件：`touchstart` 和`touchmove` 放到一个方法里处理。为什么呢？ `touchHandler` 方法定义的飞船位置变量适合下面两种情况下：当玩家触摸时，但不移动它（`touchstart`）和当手指在屏幕上开始移动（`touchmove`）：
+我们将两种事件：`touchstart` 和 `touchmove` 放到一个方法里处理。为什么呢？ `touchHandler` 方法定义的飞船位置变量适合下面两种情况下：当玩家触摸屏幕，但不移动它时（`touchstart`）和当手指在屏幕上开始移动时（`touchmove`）：
 
 ```js
 document.addEventListener("touchstart", touchHandler);
 document.addEventListener("touchmove", touchHandler);
 ```
 
-`touchHandler` 方法的代码如下：
+`touchHandler` 函数的代码如下：
 
 ```js
 function touchHandler(e) {
   if (e.touches) {
     playerX = e.touches[0].pageX - canvas.offsetLeft - playerWidth / 2;
     playerY = e.touches[0].pageY - canvas.offsetTop - playerHeight / 2;
-    output.innerHTML = "Touch: " + " x: " + playerX + ", y: " + playerY;
+    output.textContent = `Touch:  x: ${playerX}, y: ${playerY}`;
```

Comment:
could you also fix this for en-US?

Actually, we'd better to also update [this](https://github.com/end3r/JavaScript-Game-Controls/blob/636a3b85ba1be99babc769663197fb49c12b2ca0/index.html#L133)

```suggestion
    output.textContent = `Touch:\nx: ${playerX}, y: ${playerY}`;
```